config/initializers/omniauth.rb

```diff
@@ -35,13 +12,46 @@
              path_prefix: "/check-records/auth"
   end
 else
-  Rails.application.config.middleware.use OmniAuth::Strategies::OpenIDConnect, options
+  dfe_sign_in_issuer_uri = URI(ENV.fetch("DFE_SIGN_IN_ISSUER", "example"))
+
+  Rails.application.config.middleware.use OmniAuth::Builder do
+    provider :openid_connect,
+             name: :dfe,
+             callback_path: "/check-records/auth/dfe/callback",
+             client_options: {
+               host: dfe_sign_in_issuer_uri&.host,
+               identifier: ENV.fetch("DFE_SIGN_IN_CLIENT_ID", "example"),
```

Comment:
I wonder if it'd be better to set a precedent and drop the default "example' value in these options? Useful for the app to crash early (during an attempt at starting the server in the dev environment, for example) so that we can pick up any misconfiguration before getting to prod.